### PR TITLE
fix proxy setting

### PR DESCRIPTION
**Pull request by ganezasan**
# Description
Change the scripts for setting proxy

# Reasons
The initial document's script doesn't work for the current server, and we noticed `/etc/circle-installation-customizations` is just a shell script, so to set environment variables, it needs to set `export` before the environment variables.

### CODE REVIEW

File: jekyll/_cci2/proxy.adoc [hunk 1]
```diff
@@ -60,18 +61,31 @@ If you run in Amazon's EC2 service then you'll need to include `169.254.169.254`
...
+export HTTPS_PROXY=<proxy-ip:port>
+export NO_PROXY=169.254.169.254,<circleci-service-ip>,127.0.0.1,localhost,ghe.example.com
+export JVM_OPTS="-Dhttp.proxyHost=<proxy-ip> -Dhttp.proxyPort=<proxy-port> -Dhttps.proxyHost=<proxy-ip> -Dhttps.proxyPort=<proxy-port> -Dhttp.nonProxyHosts=169.254.169.254|<circleci-service-ip>|127.0.0.1|localhost|ghe.example.com"
+EOF
+) | sudo tee -a /etc/circle-installation-customizations
 
 systemctl restart replicated\*
 ```
 
+NOTE: NOTE: If the proxy requires user authentication, you will need to be added user authentication information into <proxy-ip>.
```
Comment (makotom) [inline jekyll/_cci2/proxy.adoc:74]:
```suggestion
NOTE: NOTE: If the proxy requires user authentication, you will need to add user authentication information into <proxy-ip> like below:
```

File: jekyll/_cci2/proxy.adoc [hunk 2]
```diff
@@ -60,18 +61,31 @@ If you run in Amazon's EC2 service then you'll need to include `169.254.169.254`
...
+
+```
+export HTTP_PROXY=http://<userid>:<password>@<proxy-ip>:<port>
+export HTTPS_PROXY=https://<userid>:<password>@<proxy-ip>:<port>
+```
+
 NOTE: The above is not handled by by our enterprise-setup script and will need to be added to the user data for the Services Machine startup or done manually.
 
 // not quite clear what is meant by this note... what's user data in this context?
+// If the cusotmer automatically apply the above setting, they need to add them into ServiceBox's user data.
```
Comment (makotom) [inline jekyll/_cci2/proxy.adoc:84]:
```suggestion
// If the customer wants to apply the above setting automatically, they need to add them into ServiceBox's user data.
```

**Review by makotom — APPROVED** [review]
Added some English word tuning, but in general it looks good to me! Thank you for your work. 😸

File: jekyll/_cci2/proxy.adoc (hunk 2 above)
Comment (ganezasan) [reply jekyll/_cci2/proxy.adoc:84]: Thank you so much!

**Pull request merged**